tree/dataframe/src/RDFInterfaceUtils.cxx

```diff
@@ -53,6 +54,10 @@ namespace RDF {
 // the one in the vector
 class RActionBase;
 
+bool IsImplicitMTEnabled()
+{
+   return ROOT::IsImplicitMTEnabled();
+}
```

Comment:
I don't think we need this -- can we just forward-declare `ROOT::IsImplicitMTEnabled` in `RInterface.hxx`?

Reply:
I am not sure here. I think we call `ROOT::IsImplicitMTEnabled()` explicitly.

Reply:
You can call it as long as you have a declaration for it, you don't need the definition

Reply:
it's a template!

Reply:
but that would impose an implicit dependency of headers?

Reply:
I'm pretty sure you can call `ROOT::IsImplicitMTEnabled` from the template if you just put a forward declaration for it